tests/pact/yield.repl

```diff
@@ -60,4 +93,48 @@
 (expect "step 1 executes with new yield value"
 	"someRandomStr->B"
 	(continue-pact 1 false (hash "stu-pact-id") {"a-result": "someRandomStr"}))
+
 (commit-tx)
+(begin-tx)
+
+(use yieldtest)
+
+;; set chain id to something sane
+(env-chain-data { "chain-id": "0" })
+(env-hash (hash "emily-pact-id"))
+
+;; expect successful output which primes the yiel
+(env-entity "A")
```

Comment:
No need to set any entity stuff in here ... ?

Reply:
Apparently we do because pact will just give me "skip step: string" if i don't. It's done above as well.

Reply:
Oh OK. What's going on here is the original test was for private pacts, as identified by the "A" and "B" string values in the `step`/`step-with-rollback` calls in the original. 

What you should do is revert the original test and add your pact tests as different `defpact`s that are not private.